<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="m_add">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Boid</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_cohesion">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="m_alignment">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="m_separation">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="m_mass">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="m_speed">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="m_grid"/>
   </widget>
   <widget class="QSpinBox" name="m_swarmCount">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="m_newSwarm">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>51</height>
     </size>
    </property>
    <property name="text">
     <string>Spawn!</string>
    </property>
   </widget>
   <widget class="QLabel" name="cohesion_label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cohesion</string>
    </property>
   </widget>
   <widget class="QLabel" name="alignment_label">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alignment</string>
    </property>
   </widget>
   <widget class="QLabel" name="separation_label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Separation</string>
    </property>
   </widget>
   <widget class="QLabel" name="mass_label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass/Resistance</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>360</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed/Drive</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Count:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_remove">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>470</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>  Remove Boid</string>
    </property>
   </widget>
   <widget class="QSlider" name="m_sepDist">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Separation Distance</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
